ods_ci/tests/Tests/200__monitor_and_manage/200__metrics/205_bias_metrics.robot

```diff
@@ -0,0 +1,199 @@
+    Should Be Equal As Integers    ${return_code}	 0
+    Verify Model Status    ${MODEL_ALPHA}    success
+    Install And Verify TrustyAI Service
+    Sleep    30s  msg=Give time for Trusty to see the deployment and register the pods
+    Verify Model Is Registered with TrustyAI Service
+    Send Batch Inference Data to Model     model_name=${MODEL_ALPHA}     project_name=${PRJ_TITLE}
+    ${modelId}=   Get ModelId For A Deployed Model    modelId=${MODEL_ALPHA}
+    ${modelId}    Replace String    ${modelId}    "    ${EMPTY}
+    Create Metrics request via CLI     metrics_type=dir   modelId=${modelId}    protectedAttribute="customer_data_input-3"
+    ...       favorableOutcome=0   outcomeName="predict"    privilegedAttribute=1.0    unprivilegedAttribute=0.0
+    Verify TrustyAI Metrics Exists In Observe Metrics    trustyai_dir    retry_attempts=2
+
+*** Keywords ***
+Bias Metrics Suite Setup
+    [Documentation]    Setup to configure TrustyAI metrics
+    Set Library Search Order    SeleniumLibrary
+    RHOSi Setup
+    IF    ${SETUP_MONITORING} == ${TRUE}
+        Log    Enabling and Configuring User Workload Monitoring
+        Enable User Workload Monitoring
+        Configure User Workload Monitoring
+        Verify User Workload Monitoring Configuration
+    END
+    Launch Data Science Project Main Page
+    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
+    Create S3 Data Connection    project_title=${PRJ_TITLE}    dc_name=model-serving-connection
+    ...            aws_access_key=${S3.AWS_ACCESS_KEY_ID}    aws_secret_access=${S3.AWS_SECRET_ACCESS_KEY}
+    ...            aws_bucket_name=${aws_bucket}
+    Create Model Server    token=${FALSE}    server_name=${RUNTIME_NAME}
+
+
+Enable User Workload Monitoring
+    [Documentation]    Enable User Workload Monitoring for the cluster for user-defined-projects
+    RHOSi Setup
+    ${return_code}    ${output}    Run And Return Rc And Output   oc apply -f ${MONITORING_CONFIG_FILEPATH}
+    Log To Console    ${output}
+    Should Be Equal As Integers    ${return_code}     0   msg=Error while applying the provided file
+
+Configure User Workload Monitoring
+    [Documentation]    Configure the retention period in User Workload Monitoring for the cluster.
+    ...                This period can be configured for the component as and when needed.
+    ${return_code}    ${output}    Run And Return Rc And Output   oc apply -f ${UWM_CONFIG_FILEPATH}
+    Log To Console    ${output}
+    Should Be Equal As Integers    ${return_code}     0   msg=Error while applying the provided file
+
+Verify User Workload Monitoring Configuration
+    [Documentation]    Verifies that the ALL monitoring components for user-defined-projects
+    ...    are READY in the ${DEFAULT_MONITORING_NS} namespace
+    [Tags]    Smoke
+    ...       Tier1
+    ...       OpenDataHub
+    Wait For Pods Status  namespace=${DEFAULT_MONITORING_NS}  timeout=60
```

Comment:
to improve readability, could you add the parameter with the status you-re waiting for?

Reply:
The status argument is not defined in the Openshift Library for this keyword: https://github.com/red-hat-data-services/robotframework-openshiftcli/blob/master/openshiftcli/keywords/pods.py#L116

Reply:
hm okay, ambiguous kw though.